chaos_kitten/toys_cli.py

```
@@ -0,0 +1,225 @@
+                    f.write(content)
+                    
+                try:
+                    validator = AttackProfileValidator()
+                    report = validator.validate_profile(str(temp_path))
+                    if not report.is_valid:
+                        console.print("[red]Schema validation failed:[/red]")
+                        for err in report.errors:
+                            console.print(f"  - {err}")
+                        temp_path.unlink()
+                        return
+                except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)